src/views/product-root-docs-path-landing/components/icon-card-link-grid/index.tsx

```diff
@@ -25,7 +25,7 @@ const ProductRootDocsPathLandingIconCardLinkGrid = () => {
 							icon={SUPPORTED_ICONS[iconName]}
 							productSlug={currentProduct.slug}
 							text={name}
-							url={path}
+							url={`/${currentProduct.slug}/${path}`}
```

Comment:
Self TODO 📌 Double-check this change is needed

Reply:
Double-checked, this is definitely needed! ✅ Otherwise, after clicking the `IconCardLink`, the page URL will change to something like `/[productSlug]/docs` when clicked.